base_layer/wallet/tests/output_manager_service/service.rs

```diff
@@ -362,17 +476,38 @@ fn cancel_transaction() {
 }
 
 #[test]
-fn timeout_transaction() {
+fn cancel_transaction_memory_db() {
+    cancel_transaction(OutputManagerMemoryDatabase::new());
+}
+
+#[test]
+fn cancel_transaction_sqlite_db() {
+    let db_name = format!("{}.sqlite3", random_string(8).as_str());
```

Comment:
Perhaps this snippet should be in a function as it's used in a number of places in this file

Reply:
True, lemme do that quick.